import_tracker/lazy_import_errors.py

```diff
@@ -244,7 +242,8 @@ def __delitem__(self, *_, **__):
         self._raise()
 
     def __eq__(self, *_, **__):
-        self._raise()
+        if not _is_import_time():
```

Comment:
This should return a `bool` if `_is_import_time` is true

Reply:
Ah yes, good point! Like the function `__eq__` would need to return bool?

I guess currently it would return `None` which would be falsey, so may be by we should return `False` if `_is_import_time` is `True`?